### ISSUE TITLE
Inconsistent argument name lower_is_better

### ISSUE THREAD

**Issue by benjamin-ny** [skorch-dev/skorch#462]
skorch's scoring callbacks have the `lower_is_better` argument. @mfcabrera just informed me that sklearn's [make_scorer](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html) has a `greater_is_better` argument. Since both have the same job, it would be helpful if they were named the same.

The name could be changed in skorch to make it consistent (but that would require a deprecation period) or it is just left as is (maybe arguing that skorch leanes on PyTorch here, where lower scores are better by default).

**Comment by thomasjpfan** [comment]
I agree this is a bit strange, since a skorch Scorer callback defaults to `lower_is_better=True`. It may be better to call the callback a "Metric", i.e. `EpochMetric` or `BatchMetric`.

**Comment by BenjaminBossan** [comment]
Hmm, I think it's a little late for renaming the callbacks themselves. And even if we rename the argument, I would still keep it so that lower is better by default, since that is the universal standard for neural networks.

**Comment by thomasjpfan** [comment]
I was thinking of creating a new one. It’s the English that is a big strange. Normally it is better to have a higher “score”. Since we are in the context of neural networks, we want to minimize something. 

It’s fine as is, a little confusing for new users.
